crates/sui-framework/sources/governance/validator.move

```diff
@@ -629,8 +666,8 @@ module sui::validator {
 
     /// Update protocol public key of this candidate validator
     public(friend) fun update_candidate_protocol_pubkey(self: &mut Validator, protocol_pubkey: vector<u8>, proof_of_possession: vector<u8>) {
+        // TODO[ben] - What does this mean? is this candidate / pending / active?
```

Comment:
It means candidate and should be renamed to `is_candidate`. We used them interchangeably in a few places, causing some confusions. I will change it so only candidate is used. Thanks for pointing it out!